xmake/modules/private/xrepo/quick_search/cache.lua

```diff
@@ -0,0 +1,77 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- Copyright (C) 2015-present, TBOOX Open Source Group.
+--
+-- @author      glcraft
+-- @file        cache.lua
+--
+
+import("core.base.json")
+import("core.base.global")
+import("core.package.package", {alias = "core_package"})
+import("private.action.require.impl.repository")
+
+local CACHE_PATH = path.join(global.cachedir(), "repositories_data.json")
```

Comment:
you should use `import("core.cache.globalcache")`

```lua
globalcache.cache("quick_search")
```

Reply:
OK, added in [d7f5295](https://github.com/xmake-io/xmake/pull/3214/commits/d7f529554bb6ef5c77ed36b7b6afb89ccfac0f41)